src/engraving/infrastructure/messagebox.cpp

```diff
@@ -25,7 +25,20 @@
 
 using namespace mu::engraving;
 
-MessageBox::Button MessageBox::warning(const std::string& title, const std::string& text, const std::set<Button>& buttons)
+MessageBox::Button MessageBox::warning(const std::string& title, const std::string& text, const std::set<Button>& buttons,
+                                       bool withIcon, bool withDontShowAgainCheckBox)
+{
+    return MessageBox::openDialog("WARNING", title, text, buttons, withIcon, withDontShowAgainCheckBox);
```

Comment:
It would be better to create a private enum in MessageBox and use the values from that enum instead of strings. That's more efficient and less error-prone.